spec.html

```diff
@@ -11059,6 +11060,26 @@ <h1>Examples of Automatic Semicolon Insertion</h1>
       <p>In the circumstance that an assignment statement must begin with a left parenthesis, it is a good idea for the programmer to provide an explicit semicolon at the end of the preceding statement rather than to rely on automatic semicolon insertion.</p>
     </emu-clause>
   </emu-clause>
+  <emu-clause id="sec-hazards-of-automatic-semicolon-insertion">
+    <h1>Hazards of Automatic Semicolon Insertion</h1>
+    <em>This section is non-normative.</em>
+    <p>ECMAScript programs can be written in a style with very few semicolons by relying on automatic semicolon insertion. However, as described above, semicolons are not inserted at every newline, and some of these cases which do not have an automatically inserted semicolon can be counter-intuitive or confusing.</p>
+
+    <p>As new syntactic features are added to ECMAScript, additional cases requiring explicit semicolons emerge over time. As such, consistently explicit semicolon use is recommended.</p>
```

Comment:
I feel as though this line woud be better, if it included the word `may`:

```dif
-additional cases requiring explicit semicolons emerge over time
+additional cases requiring explicit semicolons may emerge over time
```

Reply:
Is there a possible wording which could indicate that wording has been emerging over time as well?

Reply:
We have examples and proposals using this with future syntax productions; the use of `may` would not represent the fact that they are showing up in new syntax productions.

Reply:
How do you both feel about:

```diff
- additional cases requiring explicit semicolons emerge over time
+ the likelyhood of additional cases requiring explicit semicolons increases over time.
```

Reply:
To give context to my reasoning: the existing prose implies it is a foregone conclusion, which I feel is not the case.

Reply:
@keithamus the fact that the cases of this are showing up means it is not a theoretical thing/likehood is not something I would be trying to convey. The list of ASI hazards *is* growing. The likelyhood of collisions/ASI hazards increasing is true, but they are growing (and existing proposals are using syntax under the knowledge of that growth) and expected to keep growing which is what I am trying to convey.

Reply:
Sorry, I'm not conveying myself clearly here. Let me rephrase.

The existing prose implies that _for each new syntactic feature_, there will be new semicolon requirements. This is not true _for each new syntactic feature_.

Reply:
rephrased a bit